File: Beecrowd/Semana_12/atv_ 01.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int valor, x, x100 = 0, x50 = 0, x20 = 0, x10 = 0, x5 = 0, x2 = 0, x1 = 0;

    scanf("%d", &valor);

    x100 = valor / 100;
    x = valor % 100;
    // printf("valor x: %d\n", x);
    // printf("%d \n%d", x100, x);
    if(x >= 50){
        x50 = x / 50;
        x = x % 50;

    }
    // printf("valor x: %d\n", x);
    if(x >= 20){
        x20 = x / 20;
        x = x % 20;
    }
    // printf("valor x: %d\n", x);
    if(x >= 10){
        x10 = x / 10;
        x = x % 10;
        // printf("Valor de 10: %d   valor de x: %d\n", x10, x);
    }
    // printf("valor x: %d\n", x);
    if(x >= 5){
        x5 = x / 5;
        x = x % 5;
    }
    // printf("valor x: %d\n", x);
    if(x >= 2){
        x2 = x / 2;
        x = x % 2;
    }
    x1 = x;
        printf("%d\n", valor);
        printf("%d nota(s) de R$ 100,00\n", x100);
        printf("%d nota(s) de R$ 50,00\n", x50);
        printf("%d nota(s) de R$ 20,00\n", x20);
        printf("%d nota(s) de R$ 10,00\n", x10);
        printf("%d nota(s) de R$ 5,00\n", x5);
        printf("%d nota(s) de R$ 2,00\n", x2);
        printf("%d nota(s) de R$ 1,00\n", x1);
    return 0;
}

File: Beecrowd/Semana_03/atv_ 01.py
n = int(input())
v1 = n // 100
v2 = (n % 100) // 50
v3 = ((n % 100) % 50) // 20
v4 = (((n % 100) % 50) % 20) // 10
v5 = ((((n % 100) % 50) % 20) % 10) // 5
v6 = (((((n % 100) % 50) % 20) % 10) % 5) // 2
v7 = ((((((n % 100) % 50) % 20) % 10) % 5) % 2) // 1
print(n)
print(v1, 'nota(s) de R$ 100,00')
print(v2, 'nota(s) de R$ 50,00')
print(v3, 'nota(s) de R$ 20,00')
print(v4, 'nota(s) de R$ 10,00')
print(v5, 'nota(s) de R$ 5,00')
print(v6, 'nota(s) de R$ 2,00')
print(v7, 'nota(s) de R$ 1,00')